tests/test_arabic.rs

```diff
@@ -0,0 +1,10 @@
+//! Test substitution of Arabic presentation forms
+
+use unifont::i18n::preprocess_text;
+
+#[test]
+fn test_isolated_lam_alif() {
+    let input = "\u{0644}\u{0627}";
+    let expected = "\u{FEFB} ";
+    assert_eq!(preprocess_text(input), expected);
+}
```

Comment:
@Konosh93 Here's an example of a test. I copied the data from the 'net, but I'm not sure if it's correct, because the test is failing now... Please double-check!

Reply:
@mkovaxx will have once I am home, but what if you remove the space from the expected code string?

Reply:
Besides the trailing space, there was a bug in handling "la" isolated form. I tried a fix in this PR and it tested a few combinations of sentences. I will try to add more tests cases to the Arabic test file.